bot.py

```diff
@@ -166,6 +167,24 @@ async def sendErrorToAuthor(self, ctx, error=None):
 
     async def on_ready(self):
         print(f'{self.user.name} has connected to Discord! Command prefix: {COMMAND_PREFIX}')
+        print('Sending version announcements...')
+        announcer = self.get_cog('Version Announcements')
+        num_sent = await announcer.announce_latest_version()
+        print(f'Sent announcements to {num_sent} towns.')
+
+# Announcer cog
+class AnnouncerCog(commands.Cog, name='Version Announcements'):
+    def __init__(self, bot):
+        self.bot = bot;
+        self.announcer = announce.Announcer(bot, db)
+
+    async def announce_latest_version(self):
+        return await self.announcer.announce_latest_version()
```

Comment:
Check out my timer / role lookup cog. I have a neat thing in there where it has a tiny wrapper function to protect against exceptions (sending a message to the sender) and also if a string is returned, it sends that to the context. I quite like it!

Reply:
Since I don't have multiple cases yet I just stole the error logic instead of bringing the wrapper AND having to split optOut into optOutInternal just to use the wrapper